app/assets/javascripts/components/models/Reaction.js

```diff
@@ -135,6 +135,14 @@ export default class Reaction extends Element {
         break
     }
 
+    // If userText is number only, treat as normal temperature value
+    if (/^\-?\d*\.{0,1}\d{0,2}$/.test(temperature.userText)) {
```

Comment:
will have to be the same regexp as in ReactionDetails. (needs to substitute potential  ',' to '.')